db/migrate/20220530141418_add_national_institute_of_teaching_npq_lead_provider.rb

```diff
@@ -1,13 +0,0 @@
-# frozen_string_literal: true
-
-class AddNationalInstituteOfTeachingNPQLeadProvider < ActiveRecord::Migration[6.1]
-  def change
-    niot_name = "National Institute of Teaching"
-    niot_id = "3ec607f2-7a3a-421f-9f1a-9aca8a634aeb"
-    vat_chargeable = false
-
-    npq_lead_provider = NPQLeadProvider.find_or_create_by!(name: niot_name, id: niot_id, vat_chargeable:)
```

Comment:
should we check if it exists first rather than deleting the migration, same in the other migration?

Reply:
reverted and added `defined?(NPQLeadProvider)`